<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>885</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow {

	background-image: url(:/images/bg.png);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>721</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#groupbox{

	background-color: rgb(243, 243, 243);

}</string>
    </property>
    <property name="title">
     <string>Log Messages</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPlainTextEdit" name="editLog"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>1021</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#groupBox_2{

	background-color: rgb(243, 243, 243);
}</string>
    </property>
    <property name="title">
     <string>Info</string>
    </property>
    <widget class="QPushButton" name="buttonSubscribe">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>30</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Subscribe</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonConnect">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>30</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QLabel" name="User1name">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>40</y>
       <width>78</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>User1name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>59</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>You are: </string>
     </property>
    </widget>
    <widget class="QLabel" name="Topicname">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>40</y>
       <width>221</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Topicname</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>484</x>
       <y>40</y>
       <width>40</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Topic:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>84</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Speaking to:</string>
     </property>
    </widget>
    <widget class="QLabel" name="User2name">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>78</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>User2name</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="topPanel" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#topPanel {
	
	background-color: rgb(243, 243, 243);
}</string>
    </property>
    <widget class="QWidget" name="LoginIcon_2" native="true">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>71</width>
       <height>79</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#LoginIcon_2 {
	border-image: url(:/images/logo.png);
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>427</x>
       <y>10</y>
       <width>352</width>
       <height>79</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Condensed</family>
       <pointsize>48</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Super Message</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>1011</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>210</y>
      <width>311</width>
      <height>461</height>
     </rect>
    </property>
    <property name="title">
     <string>Attachments</string>
    </property>
    <widget class="QListWidget" name="Attachmentview">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>291</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Uploadfile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Upload File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="OpenAttachment">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>430</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refreshattachments">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh Attachments</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>670</y>
      <width>64</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Message:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPublish">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>690</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMessage">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>690</y>
      <width>633</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>This is a test message</string>
    </property>
   </widget>
   <widget class="QListWidget" name="Emojiwidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>720</y>
      <width>633</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonQuit">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>720</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnhome">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>720</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Return to Main</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1116</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
